src/relation_engine_server/api_versions/api_v1.py

```diff
@@ -135,3 +135,13 @@ def show_config():
         'db_name': conf['db_name'],
         'spec_url': conf['spec_url']
     })
+
+
+def _preprocess_stored_query(query_text, config):
+    """Inject some default code into each stored query."""
+    return (
+        config.get('query_prefix', '') +
+        " LET ws_ids = @ws_ids " +
```

Comment:
What happens in ws_ids isn't a bind var?

Reply:
It's always a bind var and is only set by the API. I think it's set somewhere further up. It is the array of workspace IDs that the user has access to.

Reply:
Ok. I was just thinking of queries where there's no ws involved, like ontology traversals or whatever.